protocol.c

```diff
@@ -1288,7 +1288,7 @@ enet_protocol_receive_incoming_commands (ENetHost * host, ENetEvent * event)
        }
     }
 
-    return -1;
+    return 0;
```

Comment:
This change looks good to me; it was probably a bug in https://github.com/lsalzman/enet/commit/6ef4e7d277a4f9e97e24e3f3b77835f584acd9a1.

This `return -1` used to be unreachable and the function returned -1 for error, +1 in some other case, and 0 if nothing was received explicitly (as indicated by `enet_socket_receive`). The case that the function exists because nothing happend because of too many failed attempts was never considered.

I agree that this should be equivalent to `enet_socket_receive` having resulted in `receivedLength = 0` which also means returning 0 here.